### PR TITLE
Update Rewards payout status banners

### PR DESCRIPTION
**Pull request by emerick**
Resolves https://github.com/brave/brave-browser/issues/31480

## Submitter Checklist:

- [ ] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## UI:

| Payout Will Begin | Payout Processing | Payout Completed |
| ------------------- | -------------------- | --------------------- |
|payout-will-begin-settings|payout-processing-settings|payout-completed-settings|
|payout-will-begin-ntp|payout-processing-ntp|payout-completed-ntp
|payout-will-begin-panel|payout-processing-panel|payout-completed-panel|

### CODE REVIEW

File: components/resources/rewards_strings.grdp [hunk 1]
```diff
@@ -5,13 +5,16 @@
     Check status
   </message>
   <message name="IDS_REWARDS_PAYMENT_COMPLETED" desc="">
-    Congrats! Your <ph name="MONTH">$1<ex>October</ex></ph> rewards have arrived!
+    The payout for <ph name="MONTH">$1<ex>October</ex></ph> rewards has completed.
   </message>
   <message name="IDS_REWARDS_PAYMENT_PENDING" desc="">
-    Your <ph name="MONTH">$1<ex>October</ex></ph> earnings of <ph name="AMOUNT">$2<ex>1.5 - 1.6 BAT</ex></ph> will arrive in <ph name="DAYS">$3<ex>4 days</ex></ph>.
+    The payout for <ph name="MONTH">$1<ex>October</ex></ph> rewards will begin processing by <ph name="DATE">$2<ex>October 5</ex></ph>.
```
Comment (mkarolin) [inline components/resources/rewards_strings.grdp:11]: @Miyayes "They payout **for** October rewards" sounds odd to me as we not paying for rewards, are we? Can we say "The payout **of** October rewards" or just "Your October rewards payout"?

Comment (mkarolin) [reply components/resources/rewards_strings.grdp:11]: Discussed off line.

**Review by mkarolin — APPROVED** [review]
`strings`++

**Review by zenparsing — APPROVED** [review]

File: components/resources/rewards_strings.grdp (hunk 1 above)
Comment (Miyayes) [reply components/resources/rewards_strings.grdp:11]: @mkarolin @emerick Spoke to Ryan McFadden, and the above messaging is OK and can remain as is.

**Pull request merged**